Fix several issues in webext storage docs (#43924)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/storage/storagechange/index.md

---

`StorageChange` is an object representing a change to a storage area.

> These objects are received by listeners to the {{WebExtAPIRef("storage.onChanged")}} or {{WebExtAPIRef("storage.StorageArea.onChanged")}} events.

Reviewer: ```suggestion
Objects of this type are received by {{WebExtAPIRef("storage.onChanged")}} and {{WebExtAPIRef("storage.StorageArea.onChanged")}} event listeners.
```
